deployment/charts/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
Should this match the image tag of the docker image? i.e. `fuel-core:{appVersion}`

Reply:
it can match the released version if need be